### DISCUSSION TITLE
Want to make a dropdown or TextField to Error colour when error happens on the field

### DISCUSSION THREAD

**Discussion by jecojacob** [microsoft/fluentui-blazor#1930] [Q&A]
I have a TextField and a Dropdown , after validation if user does not enter value I am showing error , But I want to highlight the field also in red colour for example..

How Can I achieve this in FluentUI-Blazor

**Comment by vnbaaij** [comment]
See https://www.fluentui-blazor.net/Forms#example for how to do this for a TextField.

This does not work yet for a FluentSelect or listbox (these are not based on `FluentInputBase` yet)
